examples/source/1.components/amp-analytics/index.html

```diff
@@ -21,31 +21,23 @@
 <!DOCTYPE html>
 <html ⚡ lang="en">
   <head>
-    <meta charset="utf-8" />
     <title>amp-analytics</title>
-    <script async src="https://cdn.ampproject.org/v0.js"></script>
     <!-- ## Setup -->
     <!--  Import the `amp-analytics` component in the header. -->
-    <script
-      async
-      custom-element="amp-analytics"
-      src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"
-    ></script>
-    <script
-      async
-      custom-element="amp-carousel"
-      src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"
-    ></script>
-    <script
-      async
-      custom-element="amp-bind"
-      src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"
-    ></script>
-    <script
-      async
-      custom-element="amp-form"
-      src="https://cdn.ampproject.org/v0/amp-form-0.1.js"
-    ></script>
+    <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js" ></script>
+    <script async custom-element="amp-animation" src="https://cdn.ampproject.org/v0/amp-animation-0.1.js" ></script>
```

Comment:
Only components should be imported that are relevant for the example. Basically: example files should be self-contained, valid AMP files.